docs/INTEGRATION.md

```diff
@@ -261,6 +261,10 @@ if docker run --rm -i hadolint/hadolint < "%d/%f"
 | grep -EC100 ':WARNING:' ; then exit 1 ; else exit 0 ; fi
 ```
 
+## Shims
+
+- [hadolint-shim](https://github.com/kornicameister/hadolint-shim)
```

Comment:
Could you add some text after the header explaining what it does? I might be helpful to others

Reply:
@lorenzo I've changed `## Shims` to `## Others`. If there will be more of shims later on,  documentation can be changed, I believe. For now my little `shim` lands in **other** just because there is nothing like that elsewhere (at least I am not aware of it) + using term `shim`, given its definition, might be a little exaggerating ;-)

Reply:
sounds good to me :)